app/models/concerns/geospatial_behavior.rb

```diff
@@ -0,0 +1,41 @@
+# Attributes and methods for image works
+module GeospatialBehavior
+  extend ActiveSupport::Concern
+  include ::CurationConcerns::GenericWorkBehavior
+  include ::CurationConcerns::BasicMetadata
+  include ::BasicGeoMetadata
+
+  included do
+   #allow members to be of any ActiveFedora::Base
+   aggregates :members, predicate: Vocab::PCDMTerms.hasMember,
+                         class_name: 'ActiveFedora::Base',
+                         type_validator: type_validator
```

Comment:
I don't think ```aggregates``` is necessary here, as this is already defined in [Hydya::PCDM](https://github.com/projecthydra-labs/hydra-pcdm/blob/bce6accb14aba03df5210321d2e52b5008658c72/lib/hydra/pcdm/models/concerns/pcdm_behavior.rb#L5-L7).